lib/rubocop/cop/style/word_array.rb

```diff
@@ -59,9 +59,21 @@ def word_regex
           cop_config['WordRegex']
         end
 
+        def autocorrect_words(word_nodes, base_line_number)
```

Comment:
It's better to make all calls downwards in the file. Put this method after `autocorrect`.